app/views/support/responsible_bodies/show.html.erb

```diff
@@ -45,29 +45,33 @@
     <% end %>
 
     <% if @responsible_body.has_virtual_cap_feature_flags_and_centrally_managed_schools? %>
-      <h2 class="govuk-heading-l govuk-!-margin-top-9">Centrally managed schools</h2>
-      <p class="govuk-body"><%= @responsible_body.name %>:</p>
-      <ul id="responsible-body-centrally-managed-stats" class="govuk-list govuk-list--bullet">
-        <li>manages ordering for <%= centrally_managing_count_or_all_schools(responsible_body: @responsible_body) %> of its schools</li>
-        <li>has <%= what_to_order_allocation_list(allocations: @virtual_cap_pools) %> available to order right now</li>
-        <li>has ordered <%= what_to_order_state_list(allocations: @virtual_cap_pools) %></li>
-      </ul>
+      <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            <h2 class="govuk-heading-l govuk-!-margin-top-9">Centrally managed schools and colleges</h2>
```

Comment:
Was this updated for consistency? As far as I'm aware there won't be any FE institutions using virtual caps.

Reply:
My understanding is there are no 'Further education providers' who are part of an RB. But an LA or trust could have the equivalent of a stand-alone sixth-form college as part of the schools they manage.

Reply:
@ralph-hawkins Technically they are all part of an RB. They just have one RB and one school, with devolved ordering and virtual caps turned off.